assemblies/assembly-pmml-models.adoc

```diff
@@ -7,7 +7,7 @@ ifdef::context[:parent-context: {context}]
 // Purpose statement for the assembly
 As a business rules developer, you can use Predictive Model Markup Language (PMML) to define statistical or data-mining models that you can integrate with your decision services in {PRODUCT}. {PRODUCT} includes consumer conformance support of PMML 4.2.1 for Regression, Scorecard, Tree, and Mining models. {PRODUCT} does not include a built-in PMML model editor, but you can use an XML or PMML-specific authoring tool to create PMML models and then integrate them with your {PRODUCT} projects.
 
-For more information about PMML, see the DMG http://dmg.org/pmml/pmml-v4-2-1.html[PMML specification].
+For more information about PMML, see the DMG http://dmg.org/pmml/pmml-v4-4-1.html[PMML specification].
```

Comment:
I would include the word "DMG" in the text of the link i.e Make "DMG PMML specification" the text of the link.
Same for the other instances of this as well.